rotkehlchen/tests/exchanges/test_okx.py

```diff
@@ -0,0 +1,802 @@
+import warnings as test_warnings
+from unittest.mock import patch
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.assets.converters import UNSUPPORTED_OKX_ASSETS, asset_from_okx
+from rotkehlchen.assets.exchanges_mappings.okx import WORLD_TO_OKX
+from rotkehlchen.constants.assets import A_ETH, A_SOL, A_USDC, A_USDT
+from rotkehlchen.errors.asset import UnknownAsset, UnsupportedAsset
+from rotkehlchen.exchanges.data_structures import AssetMovement, Trade
+from rotkehlchen.exchanges.okx import Okx
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.constants import A_XMR
+from rotkehlchen.tests.utils.mock import MockResponse
+from rotkehlchen.types import (
+    AssetAmount,
+    AssetMovementCategory,
+    Fee,
+    Location,
+    Price,
+    Timestamp,
+    TradeType,
+)
+
+
+def test_name():
+    exchange = Okx('okx1', 'a', b'a', 'a', object(), object())
+    assert exchange.location == Location.OKX
+    assert exchange.name == 'okx1'
+
+
+def test_assets_are_known(mock_okx: Okx):
+    unsupported_assets: set[str] = set(UNSUPPORTED_OKX_ASSETS)
+    common_items = unsupported_assets.intersection(set(WORLD_TO_OKX.values()))
+    assert not common_items, f'OKX assets {common_items} should not be unsupported'
+
+    okx_assets = set()
+
+    currencies = mock_okx._query_currencies()
+    for currency in currencies['data']:
+        okx_assets.add(currency['ccy'])
+
+    sorted_assets = sorted(okx_assets)
```

Comment:
why do you need to sort them?

Reply:
Oh.. I probably didn't need to do this. I think I left it in while copying this test from binance.  Sorry about that.